core/core.go

```diff
@@ -124,7 +120,7 @@ type Mounts struct {
 
 // Close calls Close() on the App object
 func (n *IpfsNode) Close() error {
-	return n.app.Stop(n.ctx)
+	return n.Process.Close()
```

Comment:
Note: If we ever remove the "process" concept, we can just stick a `fx.Shutdowner` here.